src/Submission/SubmitTSV.jsx

```diff
@@ -23,26 +23,6 @@ const SubmitTSV = ({ project, submission, onUploadClick,
   //
   const processUpload = (event) => {
     const f = event.target.files[0];
-    if (FileReader.prototype.readAsBinaryString === undefined) {
```

Comment:
i think the section you are remving here was meant to be used to support IE...

IDK if we are still going to support it, but if we do we might need to test this from a Windows env...

Reply:
Yes, I was looking at the support tables for `FileReader.readAsTextString` -- `readAsTextString` works without polyfill for IE (https://developer.mozilla.org/en-US/docs/Web/API/FileReader#Browser_compatibility)